ArduPlane/Plane.h

```diff
@@ -385,8 +385,10 @@ class Plane : public AP_HAL::HAL::Callbacks {
     AP_BattMonitor battery;
 
 #if FRSKY_TELEM_ENABLED == ENABLED
+    // Inertial Navigation
+    AP_InertialNav_NavEKF inertial_nav {ahrs};
```

Comment:
plane does have inertial_nav for quadplane, but why does frsky_telemetry need it at all? All the info is in AHRS